config/sets/symfony/symfony61.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\Name\RenameClassRector;
+use Rector\Symfony\Rector\Class_\CommandDescriptionToPropertyRector;
+
+# https://github.com/symfony/symfony/blob/6.1/UPGRADE-6.1.md
+
+return static function (RectorConfig $rectorConfig): void {
+    // @see https://symfony.com/blog/new-in-symfony-5-3-lazy-command-description
+    $rectorConfig->rule(CommandDescriptionToPropertyRector::class);
```

Comment:
Since the property is deprecated, why not use the `CommandPropertyToAttributeRector` to change it to an attribute?

Reply:
Good find! I will change it to `CommandPropertyToAttributeRector`